polling/README.md

```diff
@@ -0,0 +1,9 @@
+# Polling
+
+These samples show three different best practices for polling.
+
+1. [Frequently Polling Activity](./frequent/README.md)
+2. [Infrequently Polling Activity](./infrequent/README.md)
+3. [Periodic Polling of a sequence of activities](./periodic_sequence/README.md)
```

Comment:
```suggestion
3. [Periodic Polling of a Sequence of Activities](./periodic_sequence/README.md)
```

Inconsistent title case, but not that important

Reply:
Unresolving. "Periodic Polling of a sequence of activities" has some title words not capitalized, but the other two numeric bullets capitalize theirs.

Reply:
Fixed